Websites/perf.webkit.org/tools/process-maintenance-backlog.py

```diff
@@ -14,35 +14,36 @@ def main():
     parser.add_argument('--server-config-json', required=True, help='The path to a JSON file that specifies the perf dashboard.')
     args = parser.parse_args()
 
-    maintenace_dir = determine_maintenance_dir()
+    maintenance_dir = determine_maintenance_dir()
 
     server_config = load_server_config(args.server_config_json)
 
-    print 'Submitting results in "%s" to "%s"' % (maintenace_dir, server_config['server']['url'])
+    print(f'Submitting results in "{maintenance_dir}" to \"{server_config["server"]["url"]}\"')
```

Comment:
Nit: One set of quotes is escaped and the other isn't. This would probably be more readable without escapes overall:
```
print(f'Submitting results in "{maintenance_dir}" to "{server_config["server"]["url"]}"')
```